src/lib/structures/expr.mli

```diff
@@ -309,17 +309,26 @@ val make_triggers:
 (** [make_triggers f binders decl menv] generate multi-triggers for the
     formula [f] and binders [binders].
 
-    For axioms, we try to produce multi-triggers in the following order
-    (if we succeed to produce at least one multi-trigger, we don't try other
-    strategies):
-    1. In greedy mode:
-    - Mono-triggers without variables;
-    - Mono-triggers with variables;
-    - Multi-triggers without variables;
-    - Multi-triggers with variables.
-      2. Otherwise:
-    - Mono and multi-triggers without variables;
-    - Mono and multi-triggers with variables.
+    The following documentation describes the strategies used by SatML
+    to generate multi-triggers.
```

Comment:
If this is SatML-specific should the documentation be here?

Reply:
`FunSAT` doesn't define the same strategies (`frugal`, `normal`, ...). Instead, it uses the global option `Options.get_greedy ()` to choose between `greedy` or not and there is no `greedier mode` too. I can write a documentation for `FunSAT` if you want.

Reply:
I will try to factorize the documentation for both solvers.

Reply:
If this function is only called by SatML it is fine to document it that way — if the instantiation strategies are different I was thinking it might make more sense to document in each solver what they do.

Reply:
Or rather, to put the documentation for SatML in satml_frontend

Reply:
Finally, I don't talk about `frugal`, `normal`, ... modes because they have different names or meaning in `FunSAT`.